src/portfolio/APIClient.ts

```diff
@@ -240,6 +255,24 @@ type PortfolioStockResponse = PortfolioStock[];
 // mock portfolio while the api is not finished yet (copied from APIMocks.ts).
 // TODO: remove this
 
+const MockCorrelations: Correlations = {
+  'BMW;Apple': 1,
+  'Apple;TUM': 2,
+  'Apple;Faber': 3,
+  'TUM;BMW': 4,
+  'BMW;Faber': 5,
+  'Faber;TUM': 6,
+};
+
+const MockAnalytics: Analytics = {
+  volatility: 1.45,
+  standardDeviation: 0.12,
+  sharpeRatio: 0.45,
+  treynorRatio: 1.2,
+  debtEquity: 0.55,
+  correlations: MockCorrelations,
+};
```

Comment:
Why are the mocks in here and in the APIMocks.ts file?

Reply:
We cannot add them to APIMocks due to circular dependencies. They will be deleted from this file ASAP though.